src/main/java/br/ufes/inf/nemo/frameweb/vp/controllers/AddFrameWebConstraintToClassAttributeContextController.java

```diff
@@ -152,6 +152,8 @@ private static boolean verifyParameterType(final String value, final Class<?> pa
             "Invalid value", ViewManagerUtils.ERROR_MESSAGE);
         return true;
       }
+    } else if (String.class.isAssignableFrom(parameterType)){
+      return false;
```

Comment:
Gabriel, ao retornar `false` me parece que o laço `for` lá de `performAction()` é interrompido e ele não continua processando os demais elementos, caso o comando tenha sido aplicado a uma seleção múltipla. Algum motivo em particular para não retornar `true`? Todos os outros casos retornam `true`.

Reply:
Professor, pelo que entendi do código, é retornado `true` para os casos em que o input é inválido (por ser vazio ou pela captura de exceção ao fazer o `parse` para o tipo esperado), de forma que é executado um `continue` no `for` de `performAction()`, passando a tratar do próximo atributo selecionado.
No caso de input válido, a função `verifyParameterType()` retorna `false` e, então, o constraint é adicionado ao atributo selecionado em `performAction()`. Como nesse caso não há nenhuma exceção para o caso do `parameterType` ser `string` (me corrija se estiver errado), retornei `false` dentro do `if`.
Quanto ao caso que você citou de aplicar o comando a uma seleção múltipla, fiz o teste e aparentemente funciona. Teria como você fazer o teste para confirmar?

Reply:
Oi Gabriel, você tem razão. Ao retornar false ele prossegue o código, insere a restrição e vai para o próximo laço do `for`. Acho que as férias desligaram meu cérebro! 🤣